app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt

```diff
@@ -44,7 +45,7 @@ class FileRepository(private val context: Context) {
         val fileName = name ?: Date().toString()
         try {
             val mainDir = File(
-                context.getExternalFilesDir(Environment.DIRECTORY_PICTURES),
+                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES),
```

Comment:
let's do Dependency Injection here, shall we? would be much easier later on

Reply:
Are you sure it's working on all OS since this method is deprecated?

Reply:
It works on latest OS

Reply:
We will test on earlier version on QA